### ISSUE TITLE
Implement context-menu for 3D viewport

### ISSUE THREAD

**Issue by philippotto** [scalableminds/webknossos#5803]
## Detailed Description
The other viewports already have a context menu. For the 3D viewport the following actions could be interesting:

- all node-related context actions we already have in the other viewports
- mesh-related actions, such as:
  - jump to position
  - remove mesh
  - reload mesh
  - reveal mesh in sidebar (see #5792)
- settings (e.g., "hide wireframe")? we already have that via a settings icon, but a context menu might also be a good place.

Would make the 3D viewport easier to use.

## Context

* [ ] Specific to long-running jobs (set `jobsEnabled=true` in `application.conf`)
* [ ] Specific to webKnossos.org (set `isDemoInstance=true` in `application.conf`)

**Comment by hotzenklotz** [comment]
Related to #6300

**Comment by MichaelBuessemeyer** [comment]
@philippotto 
I have some short questions :)
Should I implement a remove or a hide action for meshes? Or both?

**Comment by philippotto** [comment]
Yes, good question! I'd say, "Hide" is more important, but if remove is easy (which I'd assume it is), let's add that too :)
